src/python/grpcio_tests/tests/reflection/_reflection_servicer_test.py

```diff
@@ -56,10 +56,11 @@ class ReflectionServicerTest(unittest.TestCase):
     # DESCRIPTOR for each message. So, the equal comparison between protobuf
     # returned by stub and manually crafted protobuf will always fail.
     def _assert_sequence_of_proto_equal(self, x, y):
-        self.assertSequenceEqual(
-            tuple(proto.SerializeToString() for proto in x),
-            tuple(proto.SerializeToString() for proto in y),
-        )
+        # self.assertSequenceEqual(
+        #     tuple(proto.SerializeToString() for proto in x),
+        #     tuple(proto.SerializeToString() for proto in y),
+        # )
+        self.assertEqual(x, y)
```

Comment:
If this comment is not applicable anymore, I will vote to remove them completely.

Reply:
Done.

Reply:
It seems comments are still there for 54-57? Can you just help to update all `_assert_sequence_of_proto_equal` with straightforward `assertEqual` in this test?